<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.trytech.tasks.automation.tests.Runner" tests="5" time="24.920" ignored="2" failures="0" timestamp="07 Jan 2017 16:24:40 GMT" skipped="0" hostname="Win" errors="1">
  <testcase name="peformBlogLinksClick" time="8.322" classname="com.trytech.tasks.automation.tests.Runner"/>
  <testcase name="teardown" time="0.000" classname="com.trytech.tasks.automation.tests.Runner">
    <error type="java.lang.RuntimeException" message="javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials c8sm38833671oia.14 - gsmtp
">
      <![CDATA[java.lang.RuntimeException: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials c8sm38833671oia.14 - gsmtp

	at com.trytech.tasks.automation.utils.CommonUtils.sendEmail(CommonUtils.java:146)
	at com.trytech.tasks.automation.tests.Runner.teardown(Runner.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:357)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials c8sm38833671oia.14 - gsmtp

	at com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:826)
	at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:761)
	at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:685)
	at javax.mail.Service.connect(Service.java:367)
	at javax.mail.Service.connect(Service.java:226)
	at javax.mail.Service.connect(Service.java:175)
	at javax.mail.Transport.send0(Transport.java:253)
	at javax.mail.Transport.send(Transport.java:124)
	at com.trytech.tasks.automation.utils.CommonUtils.sendEmail(CommonUtils.java:141)
	... 20 more
]]>
    </error>
  </testcase> <!-- teardown -->
  <testcase name="peformWebLinksClick" time="5.380" classname="com.trytech.tasks.automation.tests.Runner"/>
  <testcase name="peformYoutubeLinksClick" time="5.391" classname="com.trytech.tasks.automation.tests.Runner"/>
  <testcase name="peformSocialMediaClicks" time="5.827" classname="com.trytech.tasks.automation.tests.Runner"/>
  <testcase name="peformSmsLinksClick" classname="com.trytech.tasks.automation.tests.Runner">
    <skipped/>
  </testcase> <!-- peformSmsLinksClick -->
  <testcase name="peformSendMailClicks" classname="com.trytech.tasks.automation.tests.Runner">
    <skipped/>
  </testcase> <!-- peformSendMailClicks -->
</testsuite> <!-- com.trytech.tasks.automation.tests.Runner -->
